blueprint/src/chapters/gaussian.tex

```diff
@@ -254,11 +257,21 @@ \subsection{Gaussian measures on a finite dimensional Hilbert space}
   \widehat{\langle u, \cdot\rangle_*\mu}(x)
   &= \hat{\mu}(x \cdot u)
   \\
-  &= \exp\left(i x \langle u, m \rangle - \frac{1}{2} x^2 \langle u, \Sigma u \rangle\right)
+  &= \exp\left(i x \langle u, m \rangle - \frac{1}{2} x^2 C(u, u)\right)
   \: .
 \end{align*}
-This is the characteristic function of a Gaussian measure on $\mathbb{R}$ with mean $\langle u, m \rangle$ and variance $\langle u, \Sigma u \rangle$.
+This is the characteristic function of a Gaussian measure on $\mathbb{R}$ with mean $\langle u, m \rangle$ and variance $C(u, u)$.
 By Theorem~\ref{thm:ext_of_charFun}, $\langle u, \cdot\rangle_*\mu$ is Gaussian, hence $\mu$ is Gaussian.
+
+By Lemma~\ref{lem:IsGaussian.charFun_eq}, we deduce that for any $t \in E$ we have
+$$\exp\left(i\langle t, m \rangle - \frac{1}{2} C(t, t)\right) = \exp\left(i\langle t, m_\mu \rangle - \frac{1}{2} C_\mu(t, t)\right).$$
+In particular, for any $t$ there exists $n_t \in \mathbb{Z}$ such that
+$$i\langle t, m \rangle - \frac{1}{2} C(t, t) = i\langle t, m_\mu \rangle - \frac{1}{2} C_\mu(t, t) + 2i\pi n_t.$$
+We deduce that $n$ is a continuous map from $E$ to $\Z$, and thus must be constant because $E$ is connected. By looking at the value at $t = 0$, we deduce that for any $t$, $n_t = 0$. Looking at real and imaginary parts we obtain that for any $t$,
+$$\langle t, m \rangle = \langle t, m_\mu \rangle \quad \text{and} \quad C(t, t) = C_\mu(t, t).$$
+We immediately deduce that $m = m_\mu$. Moreover, because $C$ and $C_\mu$ are symmetric, they are characterized by their values on the diagonal. Indeed, for any $x, y$,
```

Comment:
```suggestion
We immediately deduce that $m = m_\mu$. Moreover, because $C$ and $C'_\mu$ are symmetric, they are characterized by their values on the diagonal. Indeed, for any $x, y$,
```